files/zh-cn/web/html/element/input/index.html

```diff
@@ -590,10 +633,10 @@ <h3 id="属性各论">属性各论</h3>
  <p>Valid for the <code>image</code> and <code>submit</code> input types only. See the {{HTMLElement("input/submit", "submit")}} input type for more information.</p>
  </dd>
  <dt>{{ htmlattrdef("height") }}</dt>
- <dd>如果<strong>type</strong>属性的值是 image，这个属性定义了按钮图片的高度。</dd>
+ <dd>如果 <strong>type</strong> 属性的值是 image，这个属性定义了按钮图片的高度。</dd>
  <dt>{{htmlattrdef("id")}}</dt>
  <dd>
- <p>Global attribute valid for all elements, including all the input types, it defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking. The value is used as the value of the {{htmlelement('label')}}'s <code>for</code> attribute to link the label with the form control. See the <a href="#labels">the label element</a> below.</p>
+ <p>Global attribute valid for all elements, including all the input types, it defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking. The value is used as the value of the {{htmlelement('label')}}'s for attribute to link the label with the form control. See the <a href="#labels">the label element</a> below.</p>
```

Comment:
`for` is an attribute of `<label>`:

```suggestion
 <p>Global attribute valid for all elements, including all the input types, it defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking. The value is used as the value of the {{htmlelement('label')}}'s <code>for</code> attribute to link the label with the form control. See the <a href="#labels">the label element</a> below.</p>
```